weapon-config.inc

```diff
@@ -952,7 +961,7 @@ stock IsHighRateWeapon(weaponid)
 
 stock IsMeleeWeapon(weaponid)
 {
-	return (WEAPON_UNARMED <= weaponid <= WEAPON_KATANA) || (WEAPON_DILDO <= weaponid <= WEAPON_CANE) || weaponid == WEAPON_PISTOLWHIP;
```

Comment:
Why have you removed KATANA/DILDO as melee weapons?

Reply:
WEAPON_KATANA is 8 ID and WEAPON_DILDO is 10 ID. Chainsaw is 9. So, put it in a condition there will now be a wider range check including 8, 9 and weapon IDs.

Reply:
Okay, I see that know.